### PR TITLE
Fix AGR-1948 AGR-1952 AGR-1953

### PR DESCRIPTION
**Pull request by nathandunn**
fixes:

https://agr-jira.atlassian.net/browse/AGR-1953
https://agr-jira.atlassian.net/browse/AGR-1952
https://agr-jira.atlassian.net/browse/AGR-1948

### CODE REVIEW

File: src/containers/genePage/genomeFeatureWrapper.js [hunk 1]
```diff
@@ -51,8 +51,7 @@ class GenomeFeatureWrapper extends Component {
     this.jbrowseUrl = externalJbrowseUrl;
   }
 
-
-  componentDidMount() {
+  componentDidUpdate() {
```
Comment (paaatrick) [inline src/containers/genePage/genomeFeatureWrapper.js]: The typical pattern is to make a data fetch call unconditionally in `componentDidMount` and conditionally based on checking `prevProps` in `componentDidUpdate`. See for example: https://github.com/alliance-genome/agr_ui/blob/f492f633334244bc9bf780ff3de83d58656ecc2f/src/containers/diseasePage/index.js#L34
As it stands this is firing off about half a dozen duplicate requests as the page loads which isn't desirable.

Comment (nathandunn) [reply src/containers/genePage/genomeFeatureWrapper.js]: I know.   Okay . . I'll give that a shot.

Comment (nathandunn) [reply src/containers/genePage/genomeFeatureWrapper.js]: I pushed the fix for this

**Comment by nathandunn** [comment]
> I'm not clear how these changes address point 2 in the description of AGR-1953 about adding the sequence viewer in the "Alleles and Variants" section.

Oh, add I see . . add it to the alleles section of the gene page . . . okay

**Comment by paaatrick** [comment]
Yeah please talk to Sabrina if you're unclear on the requirements.

**Comment by nathandunn** [comment]
1953 in here. FYI @sabrinatoro

image

**Comment by nathandunn** [comment]
Everything is in there now . . wasn't clear where to put it, so I emulated what we did on the alleles page .. . 
ready for review / feedback

**Pull request merged**